lib/adapters/class-vip-enterprise-search.php

```diff
@@ -62,6 +62,9 @@ public function add_aggs_to_es_query( $formatted_args ): array {
 				unset( $formatted_args['query']['match_all'] );
 				// Add bool query with filters.
 				$formatted_args['query']['bool']['filter'] = $formatted_args['post_filter']['bool']['must'];
+			} elseif ( ! empty( $formatted_args['post_filter']['bool']['must'] ) ){
```

Comment:
@jakewrfoster interested for your thoughts here - I was thinking that the query should have been modified to include a function score, but Julio is reporting that's not always the case.

Reply:
@jakewrfoster happy to provide an example, if that helps

Reply:
We went over this in review and Julio demoed the behavior - I'm good with this solution.